Tools/ros2/ardupilot_sitl/CMakeLists.txt

```diff
@@ -24,12 +24,12 @@ message(STATUS "WAF_COMMAND: ${WAF_COMMAND}")
 set(WAF_CONFIG $<$<OR:$<CONFIG:Debug>,$<CONFIG:RelWithDebInfo>>:"--debug">)
 
 add_custom_target(ardupilot_configure ALL
-  ${WAF_COMMAND} configure --board sitl --enable-dds ${WAF_CONFIG}
+  ${WAF_COMMAND} configure --board sitl --enable-dds --enable-networking ${WAF_CONFIG}
```

Comment:
--enable-networking is gone now, it is enabled by default

Reply:
Removed.